cells/std/cli/randomitems.go

```diff
@@ -0,0 +1,273 @@
+package main
+
+import (
+	"math/rand"
+	"sync"
+)
+
+type randomItemGenerator struct {
+	names          []string
+	nameIndex      int
+	organelles     []string
+	organelleIndex int
+	cells          []string
+	cellIndex      int
+	clades         []string
+	cladeIndex     int
+	descs          []string
+	descIndex      int
+	mtx            *sync.Mutex
+	shuffle        *sync.Once
+}
+
+func (r *randomItemGenerator) reset() {
+	r.mtx = &sync.Mutex{}
+	r.shuffle = &sync.Once{}
+
+	// stdMetaJson := `[{"__std_name": "name","__std_organelle": "organelle","__std_cell": "cell","__std_clade": "clade","__std_description": "A description ..."}]`
+	r.names = []string{
+		"name",
+		"default",
+		"backend",
+	}
+
+	r.cells = []string{
+		"cloud",
+		"metal",
+		"automation",
+	}
+
+	r.organelles = []string{
+		"oci-images",
+		"nomadEnvs",
+		"hydrationProfile",
+		"bitteProfile",
+		"constants",
+		"entrypoints",
+		"packages",
+		"healthChecks",
+	}
+
+	r.clades = []string{
+		"data",
+		"functions",
+		"installables",
+		"runnables",
+	}
+
+	r.descs = []string{
+		"A little weird",
+		"Bold flavor",
+		"Can’t get enough",
+		"Delectable",
+		"Expensive",
+		"Expired",
+		"Exquisite",
+		"Fresh",
+		"Gimme",
+		"In season",
+		"Kind of spicy",
+		"Looks fresh",
+		"Looks good to me",
+		"Maybe not",
+		"My favorite",
+		"Oh my",
+		"On sale",
+		"Organic",
+		"Questionable",
+		"Really fresh",
+		"Refreshing",
+		"Salty",
+		"Scrumptious",
+		"Delectable",
+		"Slightly sweet",
+		"Smells great",
+		"Tasty",
+		"Too ripe",
+		"At last",
+		"What?",
+		"Wow",
+		"Yum",
+		"Maybe",
+		"Sure, why not?",
+	}
+
+	r.shuffle.Do(func() {
```

Comment:
r.shuffle has just been unconditionally created in this very function, so it will always execute code in this anonymous function, set "done" flag internally and will never be touched again.